cls/TestCoverage/Manager.cls

```diff
@@ -176,14 +179,40 @@ Method CoverageTargetsSet(%value) As %Status [ Internal, Private ]
 	Quit $$$OK
 }
 
+/// Collect the list of PIDs used by Interoperability processes in the current namespace
+/// In order to run unit tests collecting coverage in that specific list of processes (plus the current process)
+Method GetInteropProcesses(Output tProcessIDs)
+{
+	&sql(select %DLIST(Process) into :tProcessIDs from %SYS.ProcessQuery_SS() where "User" = '_Ensemble' and Namespace = $Namespace)
+	If (SQLCODE < 0) {
+		Throw ##class(%Exception.SQL).CreateFromSQLCODE(SQLCODE,%msg)
+	}
+	if ('$LISTFIND(tProcessIDs, $Job)) {
+		set $LIST(tProcessIDs, *+1) = $Job
```

Comment:
Nit: you can just set tProcessIDs = tProcessIDs _ $ListBuild($Job); this is faster.